main.py

```diff
@@ -74,17 +79,23 @@ def score_base(self) -> Tuple[str, int, str]:
 
         return (base_source, *self.scorer.score(start_o))
 
+    def _update_seed(self) -> None:
+        rand_seed = self.random.randrange(10**18) + self.iteration
+        self.random.seed(rand_seed)
```

Comment:
No point in doing this reseeding if we don't save the new seed (which we shouldn't since it's somewhat broken, as discussed).